superset/config.py

```diff
@@ -1328,6 +1328,13 @@ def allowed_schemas_for_csv_upload(  # pylint: disable=unused-argument
 # Smaller values use less memory but may be slower for large files
 READ_CSV_CHUNK_SIZE = 1000
 
+# Maximum number of detailed errors to display in CSV upload error messages
+# When there are more errors than this limit, the system will show the first
+# CSV_UPLOAD_MAX_ERRORS_DISPLAYED errors with details and indicate how many
+# additional errors exist (e.g., "and 5 more error(s)")
+CSV_UPLOAD_MAX_ERRORS_DISPLAYED = 3
```

Comment:
@luizotavio32 I think the max errors is based on UI real state and should be a static value to avoid adding another configuration option.